features/account.feature

```diff
@@ -0,0 +1,32 @@
+Feature: Account API
+  As an authenticated user
+  I want to get account information
+  So that I can view account details
+
+  Background:
+    Given we are following "Alice"
+    And we are following "Bob"
+    And we are followed by "Alice"
+    And we are followed by "Bob"
+
+  Scenario: Get default account
+    When an authenticated "get" request is made to "/.ghost/activitypub/account"
+    Then the request is accepted with a 200
+    And the response contains our account details
+
+  Scenario: Get account by handle
+    When an authenticated "get" request is made to "/.ghost/activitypub/account?handle=@johnonolan@mastodon.xyz"
```

Comment:
Not sure if this is the best thing to do but this API does a actual remote lookup so can't test it with a fake handle

Reply:
We shouldn't add an actual external network lookup in our tests - we have a `fake-external-activitypub` service which we can use here, it's a wiremock instance and we can have it return whatever we like